ui/imports/utils/ProfileUtils.qml

```diff
@@ -7,6 +7,7 @@ import StatusQ.Core.Theme 0.1
 QtObject {
 
     readonly property int defaultDelegateHeight: 76
+    readonly property int showcaseLimit: 100
```

Comment:
there are two constants in the [backend](https://github.com/status-im/status-go/blob/fde26f3e47b279e60a62615f30db61944df3006f/protocol/identity/profile_showcase.go#L14) now. I can provide getters for them

Reply:
That would be ideal! If it is fine for you, will you be able to add this link in you ongoing pr?